### ISSUE TITLE
Training problem

### ISSUE THREAD

**Issue by wwwy-binary** [heshuting555/ReferSplat#10]
Hi, It seems that the training code needs checkpoint to start. May I know how to train the model from zero?
By the way, when the pseudo mask of other scenes will be released?
Thanks.

**Comment by heshuting555** [comment]
Thank you for your interest in our work!

You need to train original [3dgs](https://github.com/graphdeco-inria/gaussian-splatting) to obtain pretrained Gaussians for RGB rendering. 

Pseudo mask of other scenes will be released in the next few days.
